portal/src/sanity/env.ts

```diff
@@ -0,0 +1,20 @@
+export const apiVersion =
+    process.env.NEXT_PUBLIC_SANITY_API_VERSION || "2025-04-03";
+
+export const dataset = assertValue(
+    process.env.NEXT_PUBLIC_SANITY_DATASET,
```

Comment:
Tror kanskje vi må oppdatere noen miljøvariabler både her og der for at dette skal gå gjennom bygg…